### PR TITLE
Add Kubectl Install Instructions Using Winget

### PR DESCRIPTION
**Pull request by maciejporebski**
Add Winget to the install instructions for kubectl.

### CODE REVIEW

**Comment by reylejano** [comment]
@maciejporebski , please sign the [CLA](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/37192/#/?version=2 /) before this PR can be removed

More on signing the CLA: https://kubernetes.io/docs/contribute/new-content/#before-you-begin

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ The following methods exist for installing kubectl on Windows:
 If you have installed Docker Desktop before, you may need to place your `PATH` entry before the one added by the Docker Desktop installer or remove the Docker Desktop's `kubectl`.
 {{< /note >}}
 
-### Install on Windows using Chocolatey or Scoop
+### Install on Windows using Chocolatey Scoop or Winget
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:80]:
```suggestion
### Install on Windows using Chocolatey, Scoop or Winget {#install-nonstandard-package-tools}
```

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
 The following methods exist for installing kubectl on Windows:
 
 - [Install kubectl binary with curl on Windows](#install-kubectl-binary-with-curl-on-windows)
-- [Install on Windows using Chocolatey or Scoop](#install-on-windows-using-chocolatey-or-scoop)
+- [Install on Windows using Chocolatey, Scoop or Winget](#install-nonstandard-package-tools)
```
Comment (marosset) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:23]:
```suggestion
- [Install on Windows using Chocolatey, Scoop, or Winget](#install-nonstandard-package-tools)
```

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 3]
```diff
@@ -77,9 +77,9 @@ The following methods exist for installing kubectl on Windows:
 If you have installed Docker Desktop before, you may need to place your `PATH` entry before the one added by the Docker Desktop installer or remove the Docker Desktop's `kubectl`.
 {{< /note >}}
 
-### Install on Windows using Chocolatey or Scoop
+### Install on Windows using Chocolatey, Scoop or Winget {#install-nonstandard-package-tools}
```
Comment (marosset) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:80]:
```suggestion
### Install on Windows using Chocolatey, Scoop, or Winget {#install-nonstandard-package-tools}
```

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 4]
```diff
@@ -77,9 +77,9 @@ The following methods exist for installing kubectl on Windows:
...
 
-1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
+1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager, [Scoop](https://scoop.sh) command-line installer or [Winget](https://winget.run/) package manager.
```
Comment (marosset) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:82]:
```suggestion
1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager, [Scoop](https://scoop.sh) command-line installer, or [Winget](https://winget.run/) package manager.
```

**Comment by reylejano** [comment]
Preview of updated page: https://deploy-preview-37192--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-windows/

I confirm installation step from https://winget.run/pkg/Kubernetes/kubectl

/lgtm

**Comment by reylejano** [comment]
There's a second lgtm
/approve

**Pull request merged**